src/routes/components/Modal.tsx

```diff
@@ -225,15 +263,21 @@ export default function TEMPLATE_Page() {
                    function onClick() {
                     setOpen(true);
                    }
+                   
                    function onClose() {
                     setOpen(false);
                    }
                 `}
               />
 
               <GoAButton onClick={() => setOpen(true)}>Show Modal</GoAButton>
-              <GoAModal {...componentProps} open={open} onClose={onClose}>
-                <p>Your progress will not be saved.</p>
+              
+              {!isClosableChecked(componentBindings) && (
+                <GoAModal {...componentProps} open={open}>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia obcaecati id
+                molestiae, natus dicta, eaque qui iusto similique, libero explicabo eligendi eius
+                laboriosam! Repellendus ducimus officia asperiores. Eos, eius numquam.
+              
```

Comment:
Can you run the format on this page to prevent later unwanted changes?

Reply:
Done, 'Prettier' formatting applied.